tests/python_client/testcases/test_index.py

```diff
@@ -909,8 +909,7 @@ def test_drop_index_repeatedly(self, get_simple_index):
             assert len(collection_w.indexes) == 1
             collection_w.drop_index(index_name=ct.default_index_name)
             assert len(collection_w.indexes) == 0
-            collection_w.drop_index(index_name=ct.default_index_name, check_task=CheckTasks.err_res,
-                                    check_items={ct.err_code: 0, ct.err_msg: "Index doesn\'t exist."})
+            collection_w.drop_index(index_name=ct.default_index_name)
```

Comment:
Assert again after drop index is more preciseness.

Reply:
> Assert again after drop index is more preciseness.

Thanks for your advice. Modified.